makefiles/arch/riscv.inc.mk

```diff
@@ -4,8 +4,8 @@ export TARGET_ARCH ?= riscv-none-embed
 # define build specific options
 CFLAGS_CPU   = -march=rv32imac -mabi=ilp32 -mcmodel=medlow -msmall-data-limit=8
 CFLAGS_LINK  = -nostartfiles -ffunction-sections -fdata-sections
-CFLAGS_DBG  ?= -g3 -Og
-#CFLAGS_OPT  ?= -Os
+CFLAGS_DBG  ?= -ggdb -g3
+CFLAGS_OPT  ?= -Os
```

Comment:
This was previously commented out. It is worth finding out why, before we just uncomment it.

Reply:
`git blame makefiles/arch/riscv.inc.mk` shows it was like this from the start (in #8966). There's nothing in the review discussion about this particular thing. @cladmi raised this problem in #12097.

Reply:
Maybe @kaspar030 or @kenrabold can give some insight here.

Reply:
I've opened #12526 to fix this.